### ISSUE TITLE
New Deploy options are confusing compared to earlier version

### ISSUE THREAD

**Issue by XPav** [Microsoft/MixedRealityToolkit-Unity#1959]
The Deploy tab on the "Build Window" is confusing, compared to the "non-tabbed" earlier version.  

With the "Quick Options" drop down set to Hololens, but without a HL connected via USB (or an IP address), the Build all, then Install button successfully builds my app, and pops up a nice Powershell dialog as it deploys it to my local machine.   (Where it doesn't work, because it's a HL app).

So this is the same as pressing the "Install" button on the desired version.

So the current behavior is:  
1) Press Install when HL is not connected, get a Powershell dialog describing the successful installation of the .appxbundle on my local machine.  Nothing in the Unity console.
2) Press Install when a HL is connected, get a Unity "Connecting to Device Portal" box with a progress bar, and on succesful completion, in the Console say "Successfully installed blahblah.appxbundle on **Local Machine**."

So basically:
1) There is no error anymore saying "Your HL isn't plugged in".  A deploy will always succeed to somewhere.
2) The "Local Machine" is conflated with the "Device Portal On Local Machine that accesses the Hololens", which will confuse people.

In general I'd expect people to deploy AR apps Hololens for development far more often than people deploying VR apps to their local machine, and I think the dialog should reflect that.

Unity Version: 2017.4.1f1
MRTK Version:  master as of 4/9.

**Comment by StephenHodgson** [comment]
I'll look into this, and make sure to check to make sure the HoloLens is plugged in when the HoloLens sub-target is selected in the quick options.

But the build windows is also for building to PC as well, and deploying to the device was optional.  I'll look into making the process a bit more streamlined.

**Comment by XPav** [comment]
I figured that was the reason.    

Could/should the target type (Hololens vs WMR vs...) be stored with the "target IP address, username, and password"?  So one ends up with a variety of targets (Hololens or WMR), the "Pair Hololens" button goes on the right side of the dialog and only available when you have type set to Hololens".    Add special cases for "Hololens via USB" and  "WMR to Local Machine", and you've got the easy-quick options and the configurability for everyone else.

**Comment by StephenHodgson** [comment]
Merged changes in a month ago. Going to close this now.
